src/ert/scheduler/lsf_driver.py

```diff
@@ -98,13 +101,42 @@ def parse_bjobs(bjobs_output: str) -> Dict[str, Dict[str, Dict[str, str]]]:
     return {"jobs": data}
 
 
+def parse_bhist(bhist_output: str) -> Dict[str, Dict[str, int]]:
+    data: Dict[str, Dict[str, int]] = {}
+    for line in bhist_output.splitlines():
+        if line.startswith("Summary of time"):
+            assert "in seconds" in line
+        if not line or not line[0].isdigit():
+            continue
+        tokens = line.split()
+        try:
+            if len(tokens) > 10:
+                # (there are spaces in the job name, field 3)
+                data[tokens[0]] = {
+                    "pending_seconds": int(tokens[-7]),
+                    "running_seconds": int(tokens[-5]),
+                }
+            elif len(tokens) >= 6 and tokens[0] and tokens[3] and tokens[5]:
+                data[tokens[0]] = {
+                    "pending_seconds": int(tokens[3]),
+                    "running_seconds": int(tokens[5]),
+                }
+            else:
+                logger.warning(f'bhist parser could not parse "{line}"')
+        except ValueError as err:
+            logger.warning(f'bhist parser could not parse "{line}", "{err}"')
+            continue
+    return data
+
+
 class LsfDriver(Driver):
     def __init__(
         self,
         queue_name: Optional[str] = None,
         bsub_cmd: Optional[str] = None,
         bjobs_cmd: Optional[str] = None,
         bkill_cmd: Optional[str] = None,
+        bhist_cmd: Optional[str] = None,
```

Comment:
Is this driver option propagated correctly from ert config through scheduler's create_driver function?
There is a unit test for this `tests/unit_tests/scheduler/test_scheduler.py::test_scheduler_create_lsf_driver`.